src/pymor/core/pickle.py

```diff
@@ -58,8 +58,8 @@ def loads(str):
 
 else:
     from functools import partial
-    dump = partial(pickle.dump, protocol=PROTOCOL)
-    dumps = partial(pickle.dumps, protocol=PROTOCOL)
+    dump = partial(pickle.dump, protocol=protocol or PROTOCOL)
```

Comment:
There's no `protocol` to pass here. You'd need to make a full wrapper func here instead, I think. Same for `load`.

Reply:
Nice catch, never ran into that branch of course. Will propose a fix.